client/devdocs/design/index.jsx

```diff
@@ -190,6 +191,7 @@ export default class DesignAssets extends Component {
 					<DateRange readmeFilePath="date-range" />
 					<DiffViewerExample readmeFilePath="diff-viewer" />
 					<DotPager readmeFilePath="dot-pager" />
+					<Swipeable readmeFilePath="swipeabel" />
```

Comment:
The readme won't be found because we have a typo:
```suggestion
					<Swipeable readmeFilePath="swipeable" />
```